src/asynciterable/_sleep.ts

```diff
@@ -18,6 +18,10 @@ export function sleep(dueTime: number, signal?: AbortSignal) {
       resolve();
     }, dueTime);
 
+    if (unref) {
+      id.unref();
```

Comment:
How about this?
```suggestion
    if (unref && typeof id['unref'] === function) {
      id.unref();
```

Reply:
This gives an equally suspicious

```js
c&&"function"===typeof h.unref&&h.Fa();
```

and also fails the tests. I know little about UMD, but that looks like a bug in either direction. If we use `id.unref` instead, it generates code that doesn't crash, but it still ends up accessing some random property.

Reply:
Yeah my bad, I forgot to update the callsite to be subscript syntax. It should be `id['unref']()`